src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -390,6 +413,75 @@ export function transform(formConfig, form) {
     delete clonedData.providerFacility;
   }
 
+  const incidentKeys = [
```

Comment:
I wonder if it would buy us anything to make this dynamic based off of the formIteration constant?

Reply:
The source data is hardcoded to 6 incidents, so I think that would actually make it more fragile.